realm-jni/build.gradle

```diff
@@ -164,8 +164,13 @@ targets.each { target ->
 
     task "copyAndroidJni${target.name.capitalize()}"(type: Copy) {
         dependsOn "buildAndroidJni${target.name.capitalize()}"
-        from "${projectDir}/src/librealm-jni-${target.name}.so"
+        from "${projectDir}/src/librealm-jni-${target.name}-stripped.so"
         into "${projectDir}/../realm/src/main/jniLibs/${target.jniFolder}"
+        rename "librealm-jni-${target.name}-stripped.so", 'librealm-jni.so'
```

Comment:
My Gradle foo is failing me, but I' surprised that this works without a path to the project dir, but if it does, awesome :)